src/module/audio/sink_api.c

```diff
@@ -5,7 +5,11 @@
 
 #include <module/audio/sink_api.h>
 #include <module/audio/audio_stream.h>
+#ifdef MAJOR_IADSP_API_VERSION
+#define EXPORT_SYMBOL(...)
+#else
 #include <rtos/symbol.h>
+#endif
```

Comment:
I find this not obvious at all so I think most the commit message should be moved to a comment here.